composer.json

```diff
@@ -0,0 +1,92 @@
+{
+    "name": "ttn/typo3-distribution",
+    "description": "Example distribution by the TYPO3 Trainer Network",
+    "type": "project",
+    "keywords": [
+        "typo3",
+        "example"
+    ],
+    "license": "GPL-2.0-or-later",
```

Comment:
Please also add a `LICENSE` file.

Reply:
This already exists since your first initial commit.